### ISSUE TITLE
Image not captured correctly

### ISSUE THREAD

**Issue by waltercassiano** [Donaldcwl/browser-image-compression#27]
Hi 

When I upload a photo the compression works fine, if I try again with same photo the image get a background black.

**Comment by waltercassiano** [comment]
Who has been affetecd by this related issue https://bugs.chromium.org/p/chromium/issues/detail?id=1006347#c4

**Comment by bwmasotti** [comment]
Hi, I've also reproduced a similar problem, where the compressImage returns an image with a black background.

It's only happening randomly on Chrome for Android. It isn't reproduceable on Desktop Chrome or Firefox (Desktop or Android).

More details about reproducing the problem:
Android Chrome version: 77.0.3865.116
This is easily reproduceable by using the example/basic.html page alternating between one image and another to be compressed. This image seems to cause problems: https://i0.wp.com/www.techarp.com/wp-content/uploads/2017/12/Samsung-Galaxy-A8-2018-photo-sample-04.jpg?fit=4608%2C3456&ssl=1
The problem happens both when using webWorkers or not.

I've tried narrowing down the problem by logging the ImageData from the canvas that are used, and it seems to be something on the canvas that is drawn in another canvas in the function handleMaxWidthOrHeight.
